elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -1,200 +1,344 @@
+  --pfe-cta--Color--hover:        #{pfe-color(surface--darkest)};
 
-    border-color: var(--pfe-cta--main);
-    background:   var(--pfe-cta--aux);
-    color:        var(--pfe-cta--main);
+  --pfe-cta--background--focus:   #{pfe-color(surface--lightest)};
+  --pfe-cta--inner-border--focus: #{pfe-color(surface--darkest)};
+  --pfe-cta--Color--focus:        #{pfe-color(surface--darkest)};
+}
 
-    --pfe-broadcasted--color--ui-link--hover: var(--pfe-cta--main);
+/// ===========================================================================
+/// ON DARK
+/// ===========================================================================
 
-    &::after {
-      display: none;
-    }
-  }
+:host([priority="secondary"][on="dark"]),
+:host([priority="secondary"][color="alt"][on="dark"]) {
+  --pfe-cta--background:          transparent;
+  --pfe-cta--border:              #{pfe-color(surface--lightest)};
+  --pfe-cta--inner-border:        transparent;
+  --pfe-cta--Color:               #{pfe-color(surface--lightest)};
 
-  ::slotted(a:hover), // Not sure we can lump these together
-  ::slotted(a:focus) {
-    border-color: var(--pfe-cta--main--hover);
-    background:   var(--pfe-cta--main--hover);
-    color:        var(--pfe-cta--aux--hover);
-    
-    --pfe-broadcasted--color--ui-link--hover: var(--pfe-cta--aux--hover);
-  }
+  --pfe-cta--background--hover:   #{pfe-color(surface--lightest)};
+  --pfe-cta--border--hover:       #{pfe-color(surface--lightest)};
+  --pfe-cta--Color--hover:        #{pfe-color(surface--lightest--text)};
 
-  // & ::slotted(a:visited),   TBD
+  --pfe-cta--background--focus:   rgba(255, 255, 255, .95);
+  // --pfe-cta--inner-border--focus: #{pfe-color(surface--lightest--text)};
+  // --pfe-cta--Color--focus:        #{pfe-color(surface--lightest)};
 }
 
+// Style the slotted link element
+::slotted(a) {
+  position: relative;
+  display: block;
+  transition: all #{pfe-var(animation-timing)};
 
-/// ===========================================================================
-/// ON DARK
-/// ===========================================================================
-:host([on="dark"]) {
-  --pfe-cta--main:        #{pfe-color(text--on-dark)};
-  --pfe-cta--main--hover: #{pfe-color(ui-link--on-dark--hover)};
-  --pfe-cta--aux:         transparent;
-  --pfe-cta--aux--hover:  transparent;
+  line-height: inherit;
+  text-decoration: none;
+  vertical-align:  middle;
+
+  color:            #{pfe-local(Color)};
+  background-color: #{pfe-local(background)};
+
+  padding:       #{pfe-local(Padding)};
+  border-radius: #{pfe-local(BorderRadius)};
+  border:        #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} #{pfe-local(border)};
+
+  // Update the broadcasted variables, map to main color
+  --pfe-broadcasted--color--ui-link: #{pfe-local(Color)};
+  --pfe-broadcasted--color--ui-link--hover: #{pfe-local(Color--hover)};
+  --pfe-broadcasted--color--ui-link--focus: #{pfe-local(Color--focus)};
```

Comment:
Yay!!! this works great. :)

Reply:
Ok fleeting friday afternoon thought: this does not solve for this scenario:
```
    <pfe-band pfe-color="darkest"> <-- should pass through the lighter blue broadcasted var -->
        <pfe-cta ><a href="#">Default</a></pfe-cta> <--overrides it to basic link blue -->
    </pfe-cta>
```

![image](https://user-images.githubusercontent.com/456863/59951690-b0203380-9447-11e9-9c09-84d17b9b13fb.png)